projects/m1/026-is-a-string-a-palindrome/js/index.js

```diff
@@ -0,0 +1,17 @@
+const prompt = require("prompt-sync")();
+
+const string = prompt("insert string: ");
+
+const array = [];
+
+for (i=0; i<string.length;i++){
+    let toArray = array.push(string[i]);
```

Comment:
In this case I think `toArray` is **unnecessary**.
Anyway for information, here you find the **return value** of the `push` **method** ;-):
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push#return_value